server/webapp/WEB-INF/rails.new/app/views/analytics/index.html.erb

```diff
@@ -1 +1 @@
-<div class="analytics-container" data-plugin-ids="<%= @plugin_ids.to_json %>"></div>
+<div class="analytics-container" data-supported_dashboard_metrics="<%= @supported_dashboard_metrics.to_json %>"></div>
```

Comment:
Data attributes should use hyphens, not underscores.

Reply:
Oops, changing it